CoreFoundation/Base.subproj/CFPriv.h

```diff
@@ -26,14 +26,21 @@
 #include <CoreFoundation/CFSet.h>
 #include <math.h>
 
-#ifndef CF_CROSS_PLATFORM_EXPORT
-    #if !DEPLOYMENT_RUNTIME_OBJC
-        #define CF_CROSS_PLATFORM_EXPORT extern
-    #else
-        #define CF_CROSS_PLATFORM_EXPORT static __attribute__((used))
-    #endif
+#define CF_CROSS_PLATFORM_EXPORT extern
+
+#if TARGET_OS_WIN32
+  // No C99 support
+  #define _CF_RESTRICT
```

Comment:
We should conditional this over `#if defined(_MSC_VER) && !defined(__clang__)`

Reply:
I’ll write a follow-up bug to do it tomorrow.

Reply:
I’ll write a follow-up bug to do it tomorrow.